specification/vmware/resource-manager/Microsoft.AVS/preview/2021-01-01-preview/examples/Datastores_Create.json

```diff
@@ -0,0 +1,40 @@
+{
+  "parameters": {
+    "api-version": "2021-01-01-preview",
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "group1",
+    "privateCloudName": "cloud1",
+    "clusterName": "cluster1",
+    "datastoreName": "datastore1",
+    "datastore": {
+      "properties": {
+        "netAppVolume": {
+          "nfsProviderIp": "12.0.0.4",
+          "nfsFilePath": "ANFVol2"
+        }
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/{subscription-id}/resourceGroups/group1/providers/Microsoft.AVS/privateClouds/cloud1/clusters/cluster1/datastores/datastore1",
+        "name": "datastore1",
+        "properties": {
+          "provisioningState": "Succeeded"
```

Comment:
the responses should include the properties that were set in the request (exception is for secrets - which these are not)

Reply:
My bad, cameron, could you plz match this with the properties in [Get](https://github.com/Azure/azure-rest-api-specs/blob/7de2664a252a8b629b85c115168ca4012ed70025/specification/vmware/resource-manager/Microsoft.AVS/preview/2021-01-01-preview/examples/Datastores_Get.json#L15)?

Reply:
fixed in 7781bc5